packages/auth-client/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@walletconnect/auth-client",
   "description": "Auth Client for WalletConnect Protocol",
-  "version": "1.0.0",
+  "version": "1.0.0-9e643e2",
```

I guess this change is from a canary?
```suggestion
  "version": "1.0.0",
```

---

Oh yeah, will revert this since we don't need this in `main`. Thanks 👍